.github/workflows/check-vulnerability-whitesource.yml

```diff
@@ -0,0 +1,86 @@
+name: WhiteSource Vulnerability Scan
+
+on:
+  schedule:
+    - cron: '30 10 * * *'
+  repository_dispatch:
+    types: [check-vulnerability-whitesource]
+
+jobs:
+  Provision-Runners:
+    name: Provision-Runners
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_WSS_EC2_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_WSS_EC2_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Setting up runner
```

Comment:
We have stopped using provision runners for GHA. So this wont work. Not a recommended approach

Reply:
Do we have substitute for this?

Reply:
As said above. Jenkins